### DISCUSSION TITLE
C compiler cannot create executables

### DISCUSSION THREAD

**Discussion by DevVigilante** [termux/termux-packages#20895] [Q&A] [answered]
I am asking that what mistake I made that I got this error?

I executed following in GitHub Action.
```shell
./scripts/setup-ubuntu.sh
./scripts/setup-android-sdk.sh
./scripts/run-docker.sh
./build-package.sh -a arm -o output-arm -I bash
```

Error I got when `./build-package.sh -a arm -o output-arm -I bash` was executed.

```
patching file lib/readline/colors.c
patching file patchlevel.h
configure: WARNING: unrecognized options: --disable-dependency-tracking, --disable-rpath-hack, --enable-shared, --enable-static
checking build system type... x86_64-pc-linux-gnu
checking host system type... arm-unknown-linux-androideabi

Beginning configuration for bash-5.2-release for arm-unknown-linux-androideabi

checking for arm-linux-androideabi-gcc... arm-linux-androideabi-clang
checking whether the C compiler works... no
configure: error: in `/home/runner/.termux-build/bash/src':
configure: error: C compiler cannot create executables
See `config.log' for more details
Error: Process completed with exit code 1.
```

[Full Build log](https://github.com/DevAnDroidCoder/termux-packages/actions/runs/9993060252/job/27619564158#step:7:1177)

**Comment by sylirre** [comment, marked as answer]
Option `-I` can be used only when building for `com.termux`. If you set custom application package name, dependencies should be built from source rather than downloaded.

All Termux packages have hardcoded `com.termux` either as archive path component or directly in the ELF binary files.

**Reply by DevVigilante** [reply]
> Option `-I` can be used only when building for `com.termux`. If you set custom application package name, dependencies should be built from source rather than downloaded.
> 
> All Termux packages have hardcoded `com.termux` either as archive path component or directly in the ELF binary files.

Thanks for solving problem so quickly
